Documentation/Signoffs/Mounting.md

```diff
@@ -0,0 +1,123 @@
+
+# Required sections
+
+Each signoff markdown file must contain the following sections and information:
+
+## Function of the Subsystem
+
+The mounting subsystem is the design for how all of the other subsystems will be mounted to fulfill the constraints outlined in the conceptual design.
+
+## Constraints
+| No. | Constraints                                                           | Origin            |
+| --- | --------------------------------------------------------------------- | ----------------- |
+| 1  |  The overall system delay must be less than or equal to 1.4 ms  | System Requirements |
+| 2  |  System shall not impede movement and easy access throughout the room  | American Disabilities Act |
+| 3  |  The speaker output shall not be more than 0.5 m from window or wall being measured | System Requirements |
+| 4  |  System shall have a mount or multiple mounts so that it can be installed in the classroom in an unfrequented area.| System Requirements |
+| 5  |  System shall be easy to remove and setup in a different room | Broader Impacts |
+| 6  |  System shall minimize its visual distraction | Stakeholder survey |
+| 7  |  System shall follow OSHA standard 1910.95 for occupational noise exposure.| OSHA standards |
+| 8  |  System shall follow OSHA standard 1910.304-305 for wiring design and protection | OSHA standards |
+| 9  |  System shall follow the IEEE 518-1982 Guide for the Installation of Electrical Equipment to Minimize Electrical Noise Inputs to Controllers from External Sources. | IEEE standards |
+| 10  |  Overall weight of mounting subsystem cannot exceed 17 lbs.| Weight requirement |
+| 11  |  The width of the physical mount cannot exceed 49 in.| Window width measurements |
+
+
+
+
+#### 1.)	The overall system delay must be less than or equal to 1.4 ms . [Origin: System Requirements]
```

Comment:
This doesn't make sense. Why does it matter where the other mic is? Do you mean speaker? Also why is this labeled delay? Total length of the system is not clear. If the length is what matters, why is delay mentioned in the table?